deploy/sre-prometheus/100-managed-velero-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-velero-operator-alerts
+    role: alert-rules
+  name: sre-managed-velero-operator-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-managed-velero-operator-alerts
+    rules:
+    - alert: VeleroPodOfflineSRE
+      expr: kube_deployment_status_replicas_available{namespace="openshift-velero",deployment="velero"} == 0
+      for: 15m
+      labels:
+        severity: warning
+        namespace: openshift-monitoring
```

Comment:
@jewzaam if these alerts relate to a specific operator, is it better for them to be in the operators NS? or do we prefer openshift-monitoring?